sources2csr/ngs.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class LibraryStrategy(Enum):
+    """
+    Type of library strategy (sequencing type)
+    """
+    SNV = "SNV"
+    CNV = "CNV"
+
+
+class AnalysisType(Enum):
+    """
+    Type of analysis
+    """
+    WGS = "WGS"
+    WXS = "WXS"
+
+
+class NGS(BaseModel):
+    biosource_id: str
+    biomaterial_id: str
+    library_strategy: LibraryStrategy
+    analysis_type: Optional[AnalysisType]
+
+    def __hash__(self):
+        return hash(('biosource_id', self.biosource_id,
```

Comment:
So, this is not available by default? This could be simplified to `return hash(self.dict())`

Reply:
no, without this `set(ngs_objects)` returns something like "Object not hashable"
I'll try with the dictionary then!

Reply:
it's not that easy to simplify this, with the dict it gives `TypeError: unhashable type: 'dict'`

Reply:
Okay, thanks for trying.